spec/compiler/crystal/tools/doc_spec.cr

```diff
@@ -0,0 +1,43 @@
+require "../../../spec_helper"
+
+GIT_REMOTE_PATTERNS = Crystal::Doc::Generator::GIT_REMOTE_PATTERNS.keys
+private def assert_matches_pattern(url, **options : Object)
```

Comment:
Remove the `: Object` restriction and the formatter will success.

Extracted to #4145